x-pack/plugins/ml/public/application/trained_models/models_management/deployment_setup.tsx

```diff
@@ -69,12 +70,78 @@ export const StartDeploymentSetup: FC<StartDeploymentSetup> = ({ config, onConfi
     []
   );
 
-  const toggleIdSelected = threadsPerAllocationsOptions.find(
-    (v) => v.value === threadsPerAllocations
-  )!.id;
+  const disableThreadingControls = config.priority === 'low';
 
   return (
     <EuiForm component={'form'} id={'startDeploymentForm'}>
+      {config.priority !== undefined ? (
+        <EuiDescribedFormGroup
+          titleSize={'xxs'}
+          title={
+            <h3>
+              <FormattedMessage
+                id="xpack.ml.trainedModels.modelsList.startDeployment.priorityLabel"
+                defaultMessage="Priority"
+              />
+            </h3>
+          }
+          description={
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.startDeployment.priorityHelp"
+              defaultMessage="Blah blah blah"
```

Comment:
Is there text from the API docs we can use here?

Reply:
Suggestion is:

```
Select low priority for demonstrations where each model will be very lightly used.
```

Reply:
Updated in https://github.com/elastic/kibana/pull/144704/commits/3b5d1902574db4526f122f8d208c7a646f00160c